### ISSUE TITLE
Unable to find exchange rate

### ISSUE THREAD

**Issue by AnaG9696** [frappe/erpnext#37323]
### Information about bug

Hello, when I try to create new invoice in different currencies, it's unavailable, because when i try to save it comes the notification "Unable to find exchange rate for XXX to EUR for key date "2023-00-00". Please create a Currency Exchange record manually", it doesn't matter the currency or date, this notification is written in any case, I need help with that

### Module

buying, selling

### Version

Frappe Version - v13.58.12 (HEAD)
ERPNext Version - v13.53.0 (HEAD)

### Installation method

FrappeCloud

### Relevant log output / Stack trace / Full Error Message.

```shell
Unable to find exchange rate for XXX to EUR for key date "2023-10-00" Please create a Currency Exchange record manually
```

**Comment by maveezmh** [comment]
Check if you have added currency Exchange rates in Currency Exchange Doctype

**Comment by AnaG9696** [comment]
As I know, the exchange rates should be fetched automatically according to the current market rate, do we still need to add rates?

**Comment by maveezmh** [comment]
its issue with the exchange rate server. in V14 we have option to switch the server but its not there in V13

<img width="1333" alt="image" src="https://github.com/frappe/erpnext/assets/112148467/328af38c-8051-4382-bfb6-e4a448063131">

frankfurter is not fetching the exchange rates. but when i change to Exchangerate.host it works

**Comment by AnaG9696** [comment]
Thank you

**Comment by ruthra-kumar** [comment]
Fixed in v13.54.2 https://github.com/frappe/erpnext/issues/37333#issuecomment-1751726503
